command/agent/job_endpoint.go

```diff
@@ -2060,3 +2060,48 @@ func validateEvalPriorityOpt(priority int) HTTPCodedError {
 	}
 	return nil
 }
+
+func (s *HTTPServer) JobsStatusesRequest(resp http.ResponseWriter, req *http.Request) (interface{}, error) {
+	args := structs.JobsStatusesRequest{}
+	if s.parse(resp, req, &args.Region, &args.QueryOptions) {
+		return nil, nil // seems whack
```

Comment:
> // seems whack

It kinda is. I'm not sure why we return `true` on error rather than the error. The inner `parseWait` does the right thing for status codes though, which we don't when we `return nil, err`. That'd be a nice thing to fix more generally around all these little parse functions. But out of scope for this PR! :grinning: